src/escalation/jira_escalation.py

```diff
@@ -0,0 +1,226 @@
+import datetime
+from typing import Optional
+
+# import click
+# from jinja2 import Environment
+# from jinja2 import FileSystemLoader
+from simple_logger.logger import get_logger
+
+# from src.escalation.send_slack_notification import send_slack_notification
+from src.objects.jira_base import Jira
+from src.objects.slack_base import SlackClient
+
+
+LOGGER = get_logger(name=__name__)
+
+
+class Jira_Escalation:
+    """ """
+
+    def __init__(
+        self,
+        jira: Jira,
+        slack_client: SlackClient,
+        slack_channel: str,
+        default_labels: list[str],
+        additional_labels: list[str],
+        default_jira_project: str,
+    ):
+        self.default_labels = default_labels
+        self.additional_labels = additional_labels
+        self.default_project = default_jira_project
+        self.slack_client = slack_client
+        self.jira = jira
+
+        # build query and process ocp lp firewatch tickets in NOACK and ACK status
+        jira_query = ""
+        for status in ["NO ACK", "ACK"]:
+            if default_jira_project:
+                jira_query = self.build_jql(status)
+            jira_query = self.build_jql(status)
```

Comment:
is the check needed here

Reply:
not required, set default project as required parameter